Detectors/ZDC/simulation/src/Digits2Raw.cxx

```diff
@@ -494,15 +406,15 @@ void Digits2Raw::convertDigits(int ibc)
       chd.print();
     }
     UShort_t bc = mBCD.ir.bc;
-    UInt_t orbit = mBCD.ir.orbit;
+    uint32_t orbit = mBCD.ir.orbit;
     // Look for channel ID in digits and store channel (just one copy in output)
     // This is a limitation of software but we are not supposed to acquire the
     // same signal twice anyway
-    for (Int_t im = 0; im < NModules; im++) {
-      for (UInt_t ic = 0; ic < NChPerModule; ic++) {
+    for (int32_t im = 0; im < NModules; im++) {
+      for (uint32_t ic = 0; ic < NChPerModule; ic++) {
         if (mModuleConfig->modules[im].channelID[ic] == chd.id &&
             mModuleConfig->modules[im].readChannel[ic]) {
-          Int_t is = 0;
+          int32_t is = 0;
```

Comment:
this `is` is used, but usually one does e.g. `mZDC.data[im][ic].f.s00 = chd.data[is++];`